src/backend/gitalytics_api/cookies.py

```diff
@@ -37,6 +37,12 @@ def __getitem__(self, key: CookieKey):
         encrypted: str = self._request.cookies[key.value]
         json_string: str = self._fernet.decrypt(encrypted.encode()).decode()
         return json.loads(json_string)
+
+    def get(self, key: CookieKey, *, default=None):
```

Comment:
```suggestion
    def get(self, key: CookieKey, *, default=None):
```
```suggestion
    def get_or_None(self, key: CookieKey):
        pass
    def get_with_default(self, key: CookieKey, default: any):
```
Aber solange diese methoden nicht gebraucht werden, sehe ich sie eher als Dead code, und würde sie ungerne mit reinnehmen...

Reply:
1. ich habe sie erstmal auskommentiert, da sie eine hohe chance zur verwendung hat
2. deine get_or_None + get_or_default ist keine pythonische schreibweise
ps. das `: any` ist kein typ, sondern eine funktion (`any(*args)`). das richtige wäre `typing.Any`